README.md

```diff
@@ -170,6 +172,58 @@ export ASB_NGSA_MI_RESOURCE_ID=$(az identity create -g $ASB_RG_CORE -n $ASB_NGSA
 
 ```
 
+### Create managed identity for ngsa-app
+
+```bash
+
+export ASB_NGSA_MI_NAME="${ASB_DEPLOYMENT_NAME}-ngsa-id"
+
+export ASB_NGSA_MI_RESOURCE_ID=$(az identity create -g $ASB_RG_CORE -n $ASB_NGSA_MI_NAME --query "id" -o tsv)
+
+# save env vars
+./saveenv.sh -y
+
+```
+
+### Verify Firewall Assignment
+
+Verify that the developer's I.P. address is added to the firewall for the application.  This can be done by visiting the portal navigate to the Cosmos DB account in question, and select the Networking tab.  From the Networking screen for the Cosmos DB account follow the screen to add the I.P. address.  This can be accomplished by selecting the link titled  "+ Add my current IP xxx.xxx.xxx" and follow the prompts to save the I.P. address added.  
+
+Please visit this [link](https://learn.microsoft.com/en-us/azure/cosmos-db/how-to-configure-firewall) for more detailed instructions.  
+
+### Assign User to Cosmos DB
+
+```bash
+  
+  az login --tenant 'XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'
+  az account set --subscription 'XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'
+
+  # Variables
+  # built in Cosmos DB roles
+  local roles=("00000000-0000-0000-0000-000000000002") #Array of built in Cosmos roles
```

Comment:
Cosmos roles is new to me. Is there recommended documentation on how to find these role ids? Or is the one listed here the actual role id?